src/Symfony/Component/Console/Descriptor/MarkdownDescriptor.php

```diff
@@ -132,8 +156,10 @@ protected function describeApplication(Application $application, array $options
             }
 
             $this->write("\n\n");
-            $this->write(implode("\n", array_map(function ($commandName) {
-                return '* `'.$commandName.'`';
+            $this->write(implode("\n", array_map(function ($commandName) use ($description) {
+                $anchor = '#'.strtr($description->getCommand($commandName)->getName(), array(':' => ''));
```

Comment:
This allows to handle aliases. See https://github.com/symfony/symfony/pull/20896/files#diff-57586bcf27971e696e76fbcec2e17283R4 (the anchor generated for the alias)